vmdb/app/controllers/ems_infra_controller.rb

```diff
@@ -33,41 +33,56 @@ def scaling
       return
     end
 
-    @count_parameters = @stack.parameters.select { |x| x.name.include?('::count') }
+    @orig_parameters = @stack.parameters.select { |x| x.name.include?('::count') }
```

Comment:
usually it's not a good idea to obtain all records from the DB and filter them in Ruby. Even when the performance here might not be that bad, since there is tens of params, I would rather see you are using arel

so  use either where with LIKE or better AREL .e.g. like this (haven't run this so try it out)
parameters = OrchestrationStackParameter.arel_table
@stack.parameters.where(parameters[:name].matches("%::count"))